bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gems/Canopus.java

```diff
@@ -227,154 +145,99 @@ public GuideStarValidation validate(SPTarget guideStar, ObsContext ctx) {
             ).getOrElse(GuideStarValidation.UNDEFINED);
         }
 
-        private static GuideStarValidation validateVignetting(Wfs wfs, SPTarget guideStar, ObsContext ctx) {
```

Comment:
Since we are not using probe arms anymore, we can get rid of the vignetting code in validation.